src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/AuthenticatorFactoryInterface.php

```diff
@@ -38,6 +38,8 @@ public function addConfiguration(NodeDefinition $builder);
     /**
      * Creates the authenticator service(s) for the provided configuration.
      *
+     * @param array<string, mixed> $config
```

Comment:
Is it valuable? It may not help our users. Or maybe it is for static analysis tools?
We try to only add phpdocs when they bring value.